website/app/styles/pages/application/stage.scss

```
@@ -19,6 +19,7 @@
   grid-template-rows: auto auto 1fr;
   grid-template-columns: 1fr;
   width: 100%; // needed (otherwise it collapses, because is in a grid)
+  scroll-margin-top: var(--doc-page-stage-gutter-large);
```

[suggestion] my understanding is that the scroll is done on the `main` container, but because the page header has a sticky position, the page doesn't scroll to the _visual_ top. which means, for me, that this should be instead `--doc-page-header-height` (which is the same value in `px`, but semantically more correct, and resilient/future proof)

```suggestion
  scroll-margin-top: var(--doc-page-header-height);
```

---

Thanks for the suggestion! Great catch. Updated now.